### DISCUSSION TITLE
Why does git hub copilot not work?

### DISCUSSION THREAD

**Discussion by lucas-coelh** [community/community#40362] [Copilot Conversations]
### Select Topic Area

Bug

### Body

I'm logged in with an account that has access to copilot, however it doesn't work and doesn't even give me the option to activate it, I've tried deleting and downloading it again

![image](https://user-images.githubusercontent.com/80906520/204555284-23a92ead-c7c1-4154-95c1-c76642884f25.png)
![image](https://user-images.githubusercontent.com/80906520/204555224-65152129-2696-441a-8697-0660ede3d4d4.png)

**Comment by iamcjwilliams** [comment]
@Lucas-Coelh Sorry for the trouble here, what IDE are you using?
